gen3-integration-tests/tests/test_oidc_client.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+import os
+
+from utils import logger
+from gen3.auth import Gen3Auth
+from gen3.index import Gen3Index
+import utils.gen3_admin_tasks as gat
+
+
+@pytest.mark.fence
+@pytest.mark.requires_fence_client
+@pytest.mark.requires_usersync
+class TestOIDCClient:
+    def test_oidc_client_expiration(self):
+        """
+        Scenario: Test OIDC Client Expiration
+        Steps:
+            1. Create clients with various expiration times and with client_type = client_credentials
+            2. Get client_id and client_secrets after each create request and store it
+            3. Run `fence-delete-expired-clients` gen3job and check the logs for confirmation
+        """
+        clients = [
```

Comment:
let's use a dictionary of dictionaries with  first key being client name and and keys being id and secret in the nested one